pkg/v1/remote/transport/error.go

```diff
@@ -30,13 +30,23 @@ type Error struct {
 	StatusCode int
 	// The raw body if we couldn't understand it.
 	rawBody string
+	// The request that failed
+	request *http.Request
 }
 
 // Check that Error implements error
 var _ error = (*Error)(nil)
 
 // Error implements error
 func (e *Error) Error() string {
+	prefix := ""
+	if e.request != nil {
+		prefix = fmt.Sprintf("%s %s: ", e.request.Method, e.request.URL)
```

Comment:
wdyt about adding `Request: ` to the beginning (`Request: GET http://blah.com`)

Reply:
I don't love it -- seems like mostly just noise, it's pretty easy (at least for me) to recognize the $HTTP_VERB $URL pattern.